create_jenkins_job.py

```diff
@@ -124,10 +125,18 @@ def main(argv=None):
         # configure manual triggered job
         job_name = 'ci_' + os_name
         job_data['cmake_build_type'] = 'None'
+        # For the aarch64 job:
+        #  * do it nightly (because it takes a very long time to run); and
+        #  * don't email on test failures (because there are many, likely
+        #    related to qemu)
+        if os_name == 'linux-aarch64':
+            job_data['time_trigger_spec'] = '0 11 * * *'
```

Comment:
I considered this and decided that it would be more consistent to keep the `ci_` prefix because this job is configured like the other `ci_` jobs, as opposed to being configured like the other `nightly_` jobs (each of which does something special, like coverage, non-None build type, or coverage). As it is now, it'll run nightly, and also be available for manual triggering.

Do you feel strongly that this naming scheme is unclear? If so, what would you suggest for the new nightly job? Would `nightly_linux-aarch64`, with no suffix, make sense?